CHANGELOG.md

```diff
@@ -8,16 +8,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add support to get the API Server URL for both legacy and CAPI clusters
+
 ### Changed 
 
 - Upgrade ingress version to networking.k8s.io/v1beta1
 
+### Fix
+
+- Fix scrapeconfig to use secured ports for kubernetes control plane components for CAPI clusters
+- Fix scrapeconfig to proxy all calls through the API Server for CAPI clusters
+
 ## [1.24.8] - 2021-03-18
 
 ### Fix
 
 - Avoid alerting for `MatchingNumberOfPrometheusAndCluster` when a cluster is
   being deleted.
+- Fix scrapeconfig to use secured ports for kubernetes control plane components for CAPI clusters
+- Fix scrapeconfig to proxy all calls through the API Server for CAPI clusters
```

Comment:
Looks like rebase/merge error, like these lines are in two places **Unreleased** and **1.24.8**.

Reply:
Let me fix it